include/seastar/core/reactor_config.hh

```diff
@@ -29,6 +29,7 @@ namespace seastar {
 
 /// \cond internal
 struct reactor_config {
+    bool handle_sigint = true;
     bool auto_handle_sigint_sigterm = true;
```

Comment:
What's the difference between handle_sigint and auto_handle_sigint_sigterm?

The patch is just code movement, but it's moving into a public interface, we it's better if we understand the difference.

Reply:
> What's the difference between handle_sigint and auto_handle_sigint_sigterm?

The former is sub for the latter:

```c++
    if (_id == 0 && _cfg.auto_handle_sigint_sigterm) {
       if (_cfg.handle_sigint) {
          _signals.handle_signal_once(SIGINT, [this] { stop(); });
       }
       _signals.handle_signal_once(SIGTERM, [this] { stop(); });
    }
```

> The patch is just code movement, but it's moving into a public interface, we it's better if we understand the difference.

The reactor_config is not a public interface, reactor_options is